src/test/java/de/picturesafe/search/elasticsearch/ElasticsearchServiceWithoutFieldDefinitionsIT.java

```diff
@@ -138,9 +141,13 @@ public void testWithoutId() {
         assertEquals(3, result.getTotalHitCount());
         assertEquals(3, result.getResultCount());
 
-        final List<String> names = result.getSearchResultItems().stream().map(i -> (String) i.getAttribute("name")).collect(Collectors.toList());
-        assertEquals("name-1", names.get(0));
-        assertEquals("name-2", names.get(1));
-        assertEquals("name-3", names.get(2));
+        int idx = 0;
```

Comment:
Why 'idx' not 'id'?

Reply:
Because it isn't an id, it's an (array-)index.